documentation/modules/exploit/linux/ssh/mercurial_ssh_exec.md

```diff
@@ -0,0 +1,53 @@
+## Vulnerable Application
+
+[mercurial](https://www.mercurial-scm.org/downloads).
+
+This module was successfully tested against:
+
+Kali Linux, HG 4.0 and a customized hg-ssh (to simulate custom hg-ssh wrappers which have weak repo validation)
```

Comment:
I would put this line in a list. Yes its a single item list, but thats how most other docs are done